ce/cli/project.ts

```diff
@@ -11,24 +11,37 @@ import { blank } from './constants';
 import { projectFile } from './format';
 import { error, log } from './styling';
 
+class ActivationOptions {
```

Comment:
Why does this need to be here? Wouldn't it be more natural to store this in one of the activation files?

Reply:
Billy factored that out. It makes  it a bit easier to add more later, so I left it.